packages/pn-personafisica-webapp/src/components/Contacts/AddSpecialContactDialog.tsx

```diff
@@ -117,27 +117,32 @@ const AddSpecialContactDialog: React.FC<Props> = ({
       }),
   });
 
-  const initialValues = {
+  const initialValues: {
+    sender: { id: string; name: string | '' };
+    channelType: ChannelType | '';
+    s_value: string;
+  } = {
     sender: {
       id: sender.senderId,
       name: sender.senderName ?? '',
     },
-    channelType: value
-      ? channelType
-      : addressTypes.find((a) => !a.disabled && a.shown)?.id ?? ChannelType.PEC,
+    channelType: value ? channelType : '',
     s_value: value,
   };
 
   const formik = useFormik({
     initialValues,
+    initialErrors: { ...initialValues },
```

Comment:
why this?

Reply:
for set bug on button. The buttoon is active without value popolate. the paramenter in submit 'isValid' check on errors array in formik, this is usefull for popoolate this array

Reply:
this isn't the correct way because the initialErrors should not be populated with the initialValues object. Furthermore the bug seems to be not present